### PR TITLE
feat(tests): data tests

### PR DESCRIPTION
**Pull request by mkucharz**
feat(tests): data tests
feat(class): support for classes

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -25,6 +25,7 @@
     "lint": "xo src/**/*.js test/**/*.js",
```
Comment (Idered) [inline package.json]: We should clean up scripts section. It contains not used actions like `build:docs`

File: tests/e2e/class.js [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+          .then(text => {
+            console.log(text)
+            done(err)
+          })
+      })
+  })
+
+  after(function (done) {
+    deleteTestInstance(instanceName)
+      .then(() => {
```
Comment (Idered) [inline tests/e2e/class.js]: We could simplify this using http://chaijs.com/plugins/chai-as-promised/

File: tests/e2e/class.js [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+        done()
+      })
+      .catch(() => {
+        done()
+      })
+  })
+
+  it('can create a class', function (done) {
+    _class.create({name: testClassName, schema: [{"type": "string", "name": "parameter_name"}]})
+      .then(resp => {
```
Comment (Idered) [inline tests/e2e/class.js]: Let's sticks to `res` instead of `resp`. It has the same length as `err`, looks better.

File: tests/e2e/class.js [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+        console.log(resp)
```
Comment (Idered) [inline tests/e2e/class.js]: Remove console.log

File: tests/e2e/data.js [hunk 5]
```diff
@@ -0,0 +1,149 @@
...
+        expect(resp).to.be.null
+        done()
+      })
+      .catch(err => {
+        done(err)
+      })
+  })
+
+  it('can be sorted', function (done) {
+
```
Comment (Idered) [inline tests/e2e/data.js]: No `\n` here

File: tests/e2e/data.js [hunk 6]
```diff
@@ -0,0 +1,149 @@
...
+    let firstObject = null;
```
Comment (Idered) [inline tests/e2e/data.js]: We don't use semicolons.

File: tests/e2e/data.js [hunk 7]
```diff
@@ -0,0 +1,149 @@
...
+    let secondObject = null;
+
+    // Create objects
+    Promise.all([
+      data[testClassName].create({field_string: 'abcdef'}),
```
Comment (Idered) [inline tests/e2e/data.js]: `data[testClassName]` can be extracted to const

File: tests/e2e/data.js [hunk 8]
```diff
@@ -0,0 +1,149 @@
...
+      data[testClassName].create({field_string: 'cdefgh'}),
+      data[testClassName].create({field_string: 'bcdefg'})
+    ])
+    .then(objects => {
+      firstObject = objects[0]
```
Comment (Idered) [inline tests/e2e/data.js]: `const [firstObject, secondObject] = objects`

Looks better but I don't know if we have this configured in babel

File: tests/e2e/data.js (hunk 5 above)
Comment (mkucharz) [inline tests/e2e/data.js]: We should extend xo config with such rules.

File: tests/e2e/class.js (hunk 2 above)
Comment (mkucharz) [inline tests/e2e/class.js]: I added ticket to the 1.0 project https://github.com/Syncano/syncano-server-js/projects/1#card-2807179

File: package.json (hunk 1 above)
Comment (mkucharz) [inline package.json]: I added task to the project: https://github.com/Syncano/syncano-server-js/projects/1#card-2807188

**Pull request merged**